templates/shared/_charmed-kubeflow-beta-banner.html

```diff
@@ -0,0 +1,9 @@
+<div class="u-fixed-width">
+  <div class="p-notification--information is-inline">
+    <div class="p-notification__content">
+      <h5 class="p-notification__title">Beta-version of  Charmed Kubeflow 1.6 is here:</h5>
```

Comment:
I've left a comment in the brief about this, "Beta-version" doesn't look right to me.

Reply:
Updated, thanks.